### DISCUSSION TITLE
Bug in websocket section for FASTAPI docs

### DISCUSSION THREAD

**Discussion by luquitared** [fastapi/fastapi#7076] [Questions] [answered]
I was looking to implement a websocket in my application and it seems the example for websockets has a bug where the sample code is being rendered on the actual webpage. Here's the link:
https://fastapi.tiangolo.com/advanced/websockets/

**Comment by tiangolo** [comment, marked as answer]
Thanks! I think there was an error in a recent version of the docs, but it was also recently fixed. :heavy_check_mark:
